src/config.ts

```diff
@@ -146,6 +146,14 @@ const DEFAULTS = o({
         TriStart: o({
             ".*": "source_quiet",
         }),
+        TabEnter: o({
+            // "gmail.com": "mode ignore",
+        }),
+        TabLeft: o({
+            // Actually, this doesn't work because tabclose closes the current tab
+            // Too bad :/
+            // "emacs.org": "tabclose",
```

Comment:
Surely this should be `sanitise history` :D

Reply:
I was saving `sanitize` for a "DocEnd" autocommand :p